lib/awskeyring_command.rb

```diff
@@ -387,10 +387,11 @@ def token(account = nil, role = nil, code = nil) # rubocop:disable Metrics/AbcSi
 
   desc 'console ACCOUNT', I18n.t('console.desc')
   method_option :path, type: :string, aliases: '-p', desc: I18n.t('method_option.path')
+  method_option 'browser', type: :boolean, aliases: '-b', desc: I18n.t('method_option.browser'), default: false
   method_option 'no-token', type: :boolean, aliases: '-n', desc: I18n.t('method_option.notoken'), default: false
   method_option 'no-open', type: :boolean, aliases: '-o', desc: I18n.t('method_option.noopen'), default: false
   # Open the AWS Console
-  def console(account = nil) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+  def console(account = nil, browser = nil) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
```

Comment:
Because of the way autocomplete works you should leave the selection of the browser to the method option. (reverting this line pelase)

Reply:
yep 100% agree...
Have reverted